managed_vms/django_tutorial/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#	Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START docker]
+FROM gcr.io/google_appengine/python
+
+RUN apt-get update && apt-get install -y --no-install-recommends openssl ca-certificates
+
+WORKDIR /app
+RUN virtualenv /env
+ADD requirements.txt /app/requirements.txt
+RUN /env/bin/pip install -r /app/requirements.txt
+ADD . /app
+
+EXPOSE 8080
+CMD ["/env/bin/gunicorn", "-b", ":8080", "mysite.wsgi"]
```

Comment:
Can you get this to match [getting-started-python's Dockerfile](https://github.com/GoogleCloudPlatform/getting-started-python/blob/master/2-structured-data/Dockerfile#L39)?